source/Cosmos.HAL2/BlockDevice/ATAPI.cs

```diff
@@ -264,5 +272,44 @@ private void CheckForErrors()
                 //throw new Exception("ATAPI DRQ not set");
             }
         }
+        /// <summary>
+        ///  This is a good function to get max LBA I took it from https://forum.osdev.org/viewtopic.php?f=1&t=14604
```

Comment:
The summary should just be a description of the what the function does and returns
You can add the source of the code as an additional comment at the start of the function

Reply:
I added a bettter description